server/test/models/ApplicationModelTest.java

```diff
@@ -28,6 +30,41 @@ public void setUp() {
     applicationEventRepository = instanceOf(ApplicationEventRepository.class);
   }
 
+  @Test
```

Comment:
ok my bad. you were right and can remove this test actually!

Reply:
I think I will leave it Austin. This test to make sure the latest_status is carried forward if updates to the existing application row is made (today only note update is possible) This check will ensure that tomorrow say something else is updated, then the status should not be dropped.

Reply:
makes sense! we need to update the comment then cause it references the trigger this PR removes.

Reply:
sure thing!